content/docs/command-reference/get.md

```diff
@@ -65,6 +66,11 @@ name.
   download the file or directory from. The latest commit in `master` (tip of the
   default branch) is used by default when this option is not specified.
 
+- `-j <number>`, `--jobs <number>` - parallelism level for DVC to download data
+  from the remote. The default value is `4 * cpu_count()`. For SSH remotes, the
+  default is `4`. Note that the default value can be set using the `jobs` config
+  option with `dvc remote modify`. Using more jobs may speed up the operation.
```

Comment:
> Note that the default value can be set using the `jobs` config option with `dvc remote modify`.

Does this apply to `get` (and `import`)?

Reply:
I.e. does it respect the .dvc/config settings of the source repo?

Reply:
Cc @isidentical

Reply:
> I.e. does it respect the .dvc/config settings of the source repo?

I believe it should, though might be mistaken on this cc: @efiop

Reply:
In the source - yes, in the host - no. Meaning that if you run `dvc get` inside of some repo, its configs won't be used. This is consistent with all of the related commands (list/import/get). There are a few separate tasks for that, we are almost ready to get back to them, as most of the necessary pre-requisites have already been merged.

Reply:
@efiop what will be the behavior after those tasks are addressed? I'm trying to understand if we need some additional clarification here (what config is being used).

Reply:
@efiop ping ^^ I'm not sure I understood the comment about `There are a few separate tasks for that, we are almost ready to get back to them, as most of the necessary pre-requisites have already been merged.`, could you clarify it a bit please?

Reply:
> what will be the behavior after those tasks are addressed?

Don't know right now. Maybe we'll try to use host configs too or maybe we'll require explicit --config flag or config flags.

I think the confusion here is that `dvc remote modify` mentioned is talking about source repo, not the host repo.

Reply:
Yep, that's confusing. Maybe:
```suggestion
  default is `4`.  Using more jobs may speed up the operation. Note that the
  default value can be set in the source repo using the `jobs` config option of
  `dvc remote modify`.
```

Or even make that last sentence an independent block quote (at the bullet indentation level).